jest-setup.ts

```diff
@@ -24,6 +24,24 @@ if ( typeof window !== 'undefined' ) {
 			dispatchEvent: jest.fn(),
 		} ) ),
 	} );
+
+	/**
+	 * Mock `crypto.subtle.generateKey` as it's not implemented in JSDOM
+	 * https://github.com/jsdom/jsdom/issues/1612
+	 *
+	 * `crypto.subtle.generateKey` is required by `@php-wasm/web`
+	 */
+	Object.defineProperty( global.crypto, 'subtle', {
+		value: { generateKey: jest.fn() },
+	} );
```

Comment:
Here is another solution mentioned on https://github.com/jsdom/jsdom/issues/1612, and I can confirm that it works correctly and all tests pass.

```ts
import { webcrypto } from 'node:crypto';
...
Object.defineProperty( globalThis, 'crypto', {
	value: webcrypto,
} );
```